src/Appwrite/Migration/Version/V19.php

```diff
@@ -640,6 +646,21 @@ private function getFunctionCommands(Document $function): string
         return $commands;
     }
 
+    private function migrateEnumAttributeSize(): void
+    {
+        \Co\run(function () {
+            foreach ($this->documentsIterator('attributes') as $attribute) {
+                go(function ($attribute) {
```

Comment:
Considering we have to update the collection record in the metadata table, it might be dangerous to do the updates concurrently.

Reply:
We are not updating metadata table here. We are updating attributes table and collection table

Reply:
removed co routines.